plugins/org.jboss.reddeer.common/src/org/jboss/reddeer/common/util/XPathEvaluator.java

```diff
@@ -0,0 +1,152 @@
+/******************************************************************************* 
+ * Copyright (c) 2017 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.jboss.reddeer.common.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.StringWriter;
+import java.util.Iterator;
+
+import javax.xml.XMLConstants;
+import javax.xml.namespace.NamespaceContext;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Result;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathExpressionException;
+import javax.xml.xpath.XPathFactory;
+
+import org.w3c.dom.Document;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+/**
+ * XPath Evaluator
+ * 
+ * @author Lucia Jelinkova (ljelinko@redhat.com)
+ * @author Andrej Podhradsky (apodhrad@redhat.com)
+ */
+public class XPathEvaluator {
+
+	private Document doc;
+
+	/**
+	 * Constructs the XPath evaluator from the input stream and sets the
+	 * property 'namespaceAware' to a given value. The input stream is closed
+	 * once the document is built.
+	 * 
+	 * @param inputStream
+	 *            Input stream
+	 * @throws ParserConfigurationException
+	 * @throws IOException
+	 * @throws SAXException
+	 */
+	public XPathEvaluator(InputStream inputStream) throws ParserConfigurationException, SAXException, IOException {
```

Comment:
In javadoc you declare "setting property 'namespaceAware' but the method signature does not contain such parameter. Basically namespace aware is always set to true.

Could you please add constructor with parameter to set namespace awareness?